test/framework/e2e/kafka.go

```diff
@@ -26,7 +26,7 @@ type kafkaReceiver struct {
 }
 
 func (kr *kafkaReceiver) ApplicationLogs(timeToWait time.Duration) (types.Logs, error) {
-	logs, err := kr.tc.consumedLogs(kr.app.Name, loggingv1.InputNameApplication)
+	logs, err := kr.tc.consumedLogs(kr.app.Name, string(obs.InputTypeApplication))
```

Comment:
We can't necessarily because inputName != inputType.  The framework(s) has many places where for convenience we say "read from input" and use the type to equal the name but can also read from any named type.